src/gleam/string_builder.gleam

```diff
@@ -161,20 +163,13 @@ if javascript {
     "../gleam_stdlib.mjs" "length"
 }
 
+/// DEPRECATED: Use `float.to_string()` instead.
```

Comment:
Should I remove this entirely? BC break?

Reply:
Yeah, let's remove it 👍